ui/imports/shared/controls/chat/MessageReactionsRow.qml

```diff
@@ -11,22 +11,25 @@ import SortFilterProxyModel
 Row {
     id: root
 
-    required property StatusEmojiModel emojiModel
-    required property var recentEmojis
-    required property string skinColor
+    required property SortFilterProxyModel emojiModel
+    property int buttonSize
 
     signal toggleReaction(string emoji)
     signal openEmojiPopup(var parent, var mouse)
 
+    height: buttonSize
     spacing: Theme.halfPadding
     leftPadding: Theme.halfPadding
     rightPadding: Theme.halfPadding
 
-    Component.onCompleted: {
-        if (!root.recentEmojis) {
-            return
+    Connections {
+        target: root.emojiModel
+        onRecentEmojisUpdated: {
```

Comment:
This is kind of hidden dependency. Reading only the public API of that component, developer cannot know that some extra signal emitted from SFPM is assumed to be handled there. The question is why it's needed at all?

Reply:
Ok it works. I'll remove the signal